src/app/commons/Button.tsx

```diff
@@ -0,0 +1,36 @@
+import { ButtonProps } from "../types/types";
+
+const Button: React.FC<ButtonProps> = ({
+  type,
+  fullWidth,
+  children,
+  onClick,
+  secondary,
+  danger,
+  disabled,
+}) => {
+  
+  let buttonClasses =
```

Comment:
I think this can be a constant variable. Thoughts?

Reply:
I Agree , issue fixed in commit 645dc18b52e76988cc47099799ba8ffcb7515fc2.